apps/src/blockly/addons/functionBlocks.js

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
I appreciate how you moved these functions from functionEditor.js to this new file!